airflow/providers/amazon/aws/operators/s3_delete_objects.py

```diff
@@ -56,22 +59,31 @@ class S3DeleteObjectsOperator(BaseOperator):
     :type verify: bool or str
     """
 
-    template_fields = ('keys', 'bucket')
+    template_fields = ('keys', 'bucket', 'prefix')
 
     @apply_defaults
     def __init__(
             self,
             bucket,
-            keys,
+            keys=None,
+            prefix=None,
             aws_conn_id='aws_default',
             verify=None,
             *args, **kwargs):
+
+        if not bool(keys) ^ bool(prefix):
```

Comment:
Oh ya - correct. 👍  We want to **not** have both be None or set at the same time to raise an error.